sdk/src/main/java/com/horizen/box/data/NoncedBoxData.java

```diff
@@ -1,10 +1,10 @@
 package com.horizen.box.data;
 
-import com.horizen.box.NoncedBox;
+import com.horizen.box.Box;
 import com.horizen.proposition.Proposition;
 import scorex.core.serialization.BytesSerializable;
 
-public interface NoncedBoxData<P extends Proposition, B extends NoncedBox<P>> extends BytesSerializable {
+public interface NoncedBoxData<P extends Proposition, B extends Box<P>> extends BytesSerializable {
```

Comment:
Correct. I decided to put this changes into separate PR to prevent the current one growing. https://github.com/HorizenOfficial/Sidechains-SDK/pull/309